test/volume-config.sh

```diff
@@ -14,11 +14,12 @@ echo
 #
 # IMPORTANT: If these change, or a new service is introduced, it must be explicitly called out in
 # https://docs.sourcegraph.com/admin/updates/pure_docker similar to https://docs.sourcegraph.com/admin/updates/pure_docker#v3-12-5-v3-13-2-changes
+# TODO: Note instructions above (putting here so we have a diff catch as well)
```

Comment:
What does this comment mean?

Reply:
Ah that was a personal note I'm removing now.